UPGRADE-3.x.md

```diff
@@ -1,8 +1,18 @@
 UPGRADE 3.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
-=======================
+UPGRADE FROM 3.22 to 3.23
+=========================
+
+### Sonata Cache-bundle
+
+`SonataCacheBundle` was upgrade from `2.x` to `3.x`. Because of the change in the `CacheAdapterInterface`,
+technically there are few BC-break in this project:
+- `BlockEsiCache` and `BlockSsiCache` constructor signatures have changed.
+- Some methods of `BlockEsiCache`, `BlockJsCache` and `BlockSsiCache` have now return type hints.
```

Comment:
IMO, we should use "type hints" for the cases where these hints are provided through docblocks, and "type declarations" for the cases where these hints are really checked by the PHP interpreter.